test/steps/crypto/3_then.js

```diff
@@ -121,12 +121,14 @@ export function itShouldResolveToAnObjectWithThePassphraseThePrivateKeyThePublic
 		keys: { privateKey, publicKey },
 		address,
 	} = this.test.ctx;
-	const expectedObject = {
-		passphrase,
-		privateKey,
-		publicKey,
-		address,
-	};
+	const expectedObject = [
```

Comment:
probably this change is not intended?

Reply:
This change is intended as the returned value from the function is going to be an array, earlier the returned value was an object and thus the expected object would change from being an object to an array.

Reply:
ah yes, but probably better to add multiple elements check too

Reply:
@shuse2 , can you please tell me an example of existing code where we have tests for arrays?

The current test was going to just check for the object.

I had some doubts related to the regex that we are using to get the variables.

```
'a passphrase "minute omit local rare sword knee banner pair rib museum shadow juice" with private key "314852d7afb0d4c283692fef8a2cb40e30c7a5df2ed79994178c10ac168d6d977ef45cd525e95b7a86244bbd4eb4550914ad06301013958f4dd64d32ef7bc588" and public key "7ef45cd525e95b7a86244bbd4eb4550914ad06301013958f4dd64d32ef7bc588" and address "2167422481642255385L"'
```

How do we convert this given statement to use an array?

Reply:
@willclarktech , can you please guide me on how to add tests for an array?

Reply:
I think it is worth to look at how `list` command works.
Also, what I mean by test array, some edge cases like `n <= 0`.
Also, checking, both single and multiple results are as expected